modules/apps/map/map-openstreetmap/src/main/resources/META-INF/resources/js/MapOpenStreetMap.es.js

```diff
@@ -31,14 +32,21 @@ class MapOpenStreetMap extends MapBase {
 	 * @param  {Array} args List of arguments to be passed to State
 	 * @review
 	 */
-	constructor(...args) {
+	constructor(args = {...args, tileURI: defaultTileURI}) {
```

Comment:
`args = {...args,` this doesn't make much sense...

I think this should just be `args = {tileURI: defaultTileURI}`

Reply:
`args` don't contain the `tileURI` by default, so I have to add them to the `args` or `this` somewhere. 

If I do `constructor(args = {tileURI: defaultTileURI}) {}` the `args.tileURI` is undefined.

If I analyze `args = {tileURI: defaultTileURI}` it sets the default value of `args` to be `{tileURI: defaultTileURI}`, but `args` is never undefined, so it will never receive that default value, and instead override our `args.tileURI` property with an object that doesn't contain it.